pkg/driver/node.go

```diff
@@ -516,8 +516,7 @@ func (d *nodeService) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoReque
 	zone := d.metadata.GetAvailabilityZone()
 
 	segments := map[string]string{
-		TopologyKey:          zone,
-		WellKnownTopologyKey: zone,
+		TopologyKey: zone,
```

Comment:
this part I don't understand, if the volume gets migrated and its TopologyKey gets translated to WellKnownTopologyKey, how will it get scheduled to any node if all nodes only have TopologyKey? the tests passed so clearly it got scheduled

Reply:
IMO WellKnownTopologyKey labels are still added to nodes by kubelet. And in future, when there is no cloud provider in kubelet, I don't know who adds them... external cloud provider?

Reply:
Scratch the previous comment. Scheduler / its PV topology code translates in-tree volumes to CSI:

https://github.com/kubernetes/kubernetes/blob/2bbc0e3c54fdf11b0006efe7f65f6a416ea15d32/pkg/controller/volume/scheduling/scheduler_binder.go#L816